tests/test_video.py

```diff
@@ -568,10 +574,18 @@ def test_concat_videos_mixed_formats(self, reset_db: None, tmp_path: Path) -> No
             )
 
     def _validate_splitter_segments(
-        self, base: pxt.Table, segments_view: pxt.Table, overlap: float, min_segment_duration: float
+        self,
+        base: pxt.Table,
+        segments_view: pxt.Table,
+        overlap: float | None,
+        min_segment_duration: float,
+        expected_durations: list[float] | None = None,
+        eps: float = 0.0,
```

Comment:
Document the meaning of `eps`? (I can infer it by reading the code, but it's not exactly self-documenting)